### PR TITLE
 [SPARK-8111] [SparkR] SparkR shell should display Spark logo and version banner on startup. 

### PR DESCRIPTION
**Pull request by aloknsingh**
spark version is taken from the environment variable SPARK_VERSION

### CODE REVIEW

**Comment by andrewor14** [comment]
nice! @shivar

**Comment by andrewor14** [comment]
ok to test. @aloknsingh Can you post a screenshot of what the logo looks like?

File: R/pkg/inst/profile/shell.R [hunk 1]
```diff
@@ -29,5 +30,18 @@
   sqlContext <- SparkR::sparkRSQL.init(sc)
   assign("sqlContext", sqlContext, envir=.GlobalEnv)
   cat("\n Welcome to SparkR!")
```
Comment (shivaram) [inline R/pkg/inst/profile/shell.R]: We can remove `SparkR` from here and just make this `Welcome to`. That way the entire sentence will read `Welcome to Spark`

File: R/pkg/inst/profile/shell.R [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 .First <- function() {
   home <- Sys.getenv("SPARK_HOME")
+  sparkVer <- Sys.getenv("SPARK_VERSION")
```
Comment (shivaram) [inline R/pkg/inst/profile/shell.R]: I'm not sure when this environment variable is set. We can instead call the java method `sc.version` after creating the spark context. To do that you can do something like 
```
sparkVer <- SparkR:::callJMethod(sc, "version")
```

**Comment by shivaram** [comment]
Thanks @aloknsingh for the change. I had some minor comments inline. Also to reply to your question from the JIRA I'd say we can just keep the ASCII art as `Spark` right now.

**Comment by aloknsingh** [comment]
@sivaram: I modified the code as per your suggestions.

@andrewor14: Here is the screenshot.
spark-8111_sparklogo

**Comment by aloknsingh** [comment]
Jenkins. Please retest.

**Comment by shivaram** [comment]
@aloknsingh Actually the tests passed if you look at the Spark QA message (above the jenkins one) -- the jenkins plugin has an issue where it reports failures for some builds.

**Comment by andrewor14** [comment]
retest this please

**Comment by shivaram** [comment]
LGTM. Will merge once tests pass

**Comment by andrewor14** [comment]
Hm, this should have just run the R tests.

**Comment by shivaram** [comment]
Yeah I have no idea why that changed -- @JoshRosen did the limited unit test running thing change again ?

**Comment by JoshRosen** [comment]
It looks like the pull request builder somehow thinks that the `sql` module changed.

**Comment by shivaram** [comment]
hmm maybe it compared this PR to a different cached git commit ? I've seen that produce warnings for new classes added etc. before

**Comment by JoshRosen** [comment]
There might have been a race if a PR that changed SQL was committed between the time that the PRB was triggered with a merge commit and the time that we fetched origin/master to do a comparison for changed files.  We can probably fix this by using a different base / SHA to compute the diff.

**Pull request closed without merging**